README.md

```diff
@@ -1,11 +1,283 @@
+	1. [Setting default player controls’ tint color](#t11)
+	2. [Playing with AutoPlay on/off](#t12)
+	3. [Playing Muted](#t13)
+	4. [Disabling HLS (or forcing MP4 playback)](#t14)
+12. [Tutorial 2 – Customizing the Default Controls UX](#t2)
+	1. [Hiding Various Controls buttons](#t21)
+	2. [Closed Captioning / SAP Settings button](#t22)
+	3. [Using the 4 Custom Sidebar buttons](#t23)
+	4. [Setting the LIVE indicator’s tint color](#t24)
+13. [Tutorial 3 – Observing the Player](#t3)
+	1. [Pausing or Resume Playback and Seeking](#t31)
+	2. [Looping Playback](#t32)
+	3. [LIVE, VOD, or 360°?](#t33)
+	4. [Manually Hooking up Previous or Next Videos](#t34)
+14. [Tutorial 4 – Error Handling in the SDK](#t4)
+	1. [SDK Initialization Errors](#t41)
+	2. [Player Initialization Errors](#t42)
+	3. [Restricted Videos](#t43)
+15. [Specific Notes for tvOS Apps](#tvos)
+16. [Tutorial 5 – Playing videos on tvOS](#t5)
+17. [Next Steps – Getting O2 Video/Playlist IDs into your apps](#go2)
+18. [Next Steps – Controlling Ads via your O2 Portal Account](#co2)
+
+
+## What is the O2 Mobile SDK? <a name="what"></a>
+The O2 Mobile SDK (OMSDK or SDK) is a native iOS SDK with the sole purpose for playing and monetizing videos from the Oath O2 video platform in your app. The OMSDK is written in Swift and is delivered as a framework. You can include this in your app projects either via CocoaPods or Carthage. Currently, Swift Package Manager not supported on iOS or tvOS.
+
+As part of playing videos, the OMSDK also handles video ads (pre-roll, mid-roll, and post-roll) and associated videos and ads playback and performance analytics. Analytics are focused on tracking what is played, how far it is played (e.g., deciles, quartiles), and details about the actual device or network. For more details on the analytics supported or to access the analytics data, you will work with the [Video Support Team](mailto:video.support@oath.com) to build reports that focus specifically on your app’s video and ads performance.
+
+The SDK includes a complete default video player controls UX (user experience), which includes a limited albeit robust set of customization options. The controls implementation is fully open source, and the SDK architecture allows for you to include your own fully customized controls UX, should you not be interested in the built-in default one.
+
+## Main SDK Features <a name="features"></a>
+* Playback of one or more individual videos or a single playlist of videos
+* Video playback of VOD (video on demand), 360°, and LIVE streaming video types
+* Supports either .mp4 or .m3u8 (HLS) formats
+* Video ads (VAST support of .mp4 format only)
+* Tap an ad to access ad URL (more information) via an in-app-browser
+* Full video and ads analytics
+* Default video player controls UX (full source code open sourced)
+* HLS support for multiple closed captioning (CC) and SAP audio languages
+* Mute/Unmute support
+* Automatic filtering of geo-restricted videos
```

Comment:
Not on client side.

Reply:
breakdown between what SDK specifically does and what our m/s does is irrelevant to audience.  it's a feature.

Reply:
This document is about client SDK primarly - we are not doing this on client side - that's why nobody is intersted in this detail